WordPress/src/main/res/values-night/styles.xml

```diff
@@ -138,11 +138,6 @@
         <item name="android:navigationBarColor">@android:color/black</item>
     </style>
 
-    <style name="WordPress.ZenDesk.Toolbar" parent="Widget.MaterialComponents.Toolbar.Surface">
```

Comment:
Praise (❤️): I really love what you did there, extracting the all the `styles_toolbar.xml` files. Doing that makes things so much more structured. Kudos!